components/nimbus/src/stateful/nimbus_client.rs

```diff
@@ -704,6 +719,23 @@ impl NimbusClient {
         }
         Ok(())
     }
+
+    #[cfg(test)]
+    pub fn get_metrics_handler(&self) -> &TestMetrics {
```

Comment:
Is this called? I don't see any usage in the patch.

Reply:
Whoopsie, they were supposed to be